pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: collection
-version: 1.14.4-dev
+version: 1.15.0
```

Comment:
On 2nd thought, I'd hold off doing this. Copy-paste the classes and then deploy.

We really want an upper bound on the SDK constraint so folks don't get this package w/ the SDK that doesn't have these classes.

Reply:
There is an upper-bound on the SDK constraint of <2.0.0, and we will not remove the classes from Dart before 2.0.0. 

Is the problem that some 2.0.0-rcX will count as <2.0.0?